Source/Dafny/AST/ASTExport/Program.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System;
```

Comment:
Should these changes be in this PR? This doesn't seem to relate to the PR title, and the PR was already quite large.

Reply:
No, I mistakenly continued working on the PR branch.  I've reset it now.